### ISSUE TITLE
Add a button "Fetch all repo"

### ISSUE THREAD

**Issue by mattpltt** [desktop/desktop#13700]
### Describe the feature or problem you’d like to solve:

While working with many repositories, it's a long operations to fetch each repo (and not wait for the auto-fetch).

### Proposed solution

Add a new button "Fetch all repo" that can force fetch for each repo listed.

### Sometyhing like this:

![github_desktop](https://user-images.githubusercontent.com/285024/150105194-66768da3-e51c-4ad5-b447-7fc3aa4426c2.png)

**Comment by steveward** [comment]
Thanks for the feedback @mattpltt. GitHub Desktop has background fetching at set intervals for all repos and also every time you switch repositories. Is there a specific use-case you have for wanting to force a fetch at a greater interval?

**Comment by mattpltt** [comment]
Yes: usually i work with a large software solution that is linked at many different repositories. So it can be very useful beeing able to force the fetch operations for each repo mapped in one single click. To date, with the current Github Desktop version, i need to switch each single repo or wait.

**Comment by steveward** [comment]
From  https://github.com/desktop/desktop/issues/16750:

>The dots indicating something has changed do not become active for a significant amount of time after a change. They eventually do become highlighted (the changed dot), but I would like for this to be either way more responsive or provide a refresh button so the repository statuses can be updated on request.

**Comment by SynapseScribe** [comment]
Which is the interval the background fetching is done?

**Comment by steveward** [comment]
@fusionneur there's an explanation [here](https://github.com/desktop/desktop/issues/4950#issuecomment-397694857).
